### PR TITLE
fix chart parse image failed.

### PR DESCRIPTION
**Pull request by yyf1986**
fix_chart_1431
default like:
```
- name: ***
  image: ***
```
support chart image like 
```
- image: ***
  name: ***
```

### CODE REVIEW

File: pkg/buildimage/chart.go [hunk 1]
```diff
@@ -113,12 +113,18 @@ func (c Chart) GetImages() ([]string, error) {
...
 		return shaReg.ReplaceAllString(a, "")
 	}
+	// add del -image: and image:
+	delImageReg := regexp.MustCompile(`-{0,1}image:`)
+	delImage := func(str string) string {
+		return delImageReg.ReplaceAllString(str, "")
+	}
 
 	for _, v := range content {
 		if delLF(v) != "" { // Text has content
```
Comment (fanux) [inline pkg/buildimage/chart.go:123]: if delLF(v) == "" { continue } ...

File: pkg/buildimage/chart.go [hunk 2]
```diff
@@ -113,12 +113,18 @@ func (c Chart) GetImages() ([]string, error) {
 	delImageSha := func(a string) string {
 		return shaReg.ReplaceAllString(a, "")
 	}
+	// add del -image: and image:
+	delImageReg := regexp.MustCompile(`-{0,1}image:`)
+	delImage := func(str string) string {
+		return delImageReg.ReplaceAllString(str, "")
+	}
 
 	for _, v := range content {
```
Comment (fanux) [inline pkg/buildimage/chart.go:122]: Use a parse function, and add some unitest for this function:

func parseImages(content) []string {}

File: pkg/buildimage/chart.go (hunk 1 above)
Comment (yyf1986) [reply pkg/buildimage/chart.go:123]: what's meaning?

File: pkg/buildimage/chart.go (hunk 2 above)
Comment (yyf1986) [reply pkg/buildimage/chart.go:122]: Need to write a unit test?

Comment (fanux) [reply pkg/buildimage/chart.go:122]: yes, unit test for this function is important

File: pkg/buildimage/chart.go (hunk 1 above)
Comment (fanux) [inline pkg/buildimage/chart.go:123]:
```suggestion
		if delLF(v) == "" { 
		    continue
		}
		for _,s := range ....
		```

**Comment by fanux** [comment]
@yyf1986 rebase

**Comment by yyf1986** [comment]
resubmit

**Pull request closed without merging**